pkg/opssight/perceivers.go

```diff
@@ -194,6 +302,50 @@ func (p *SpecConfig) ImagePerceiverService() *components.Service {
 	return p.perceiverService(p.names["image-perceiver"])
 }
 
+// ArtifactoryPerceiverService creates a service for the Artifactory perceiver
+func (p *SpecConfig) ArtifactoryPerceiverService() *components.Service {
+	name := p.names["artifactory-perceiver"]
+	service := components.NewService(horizonapi.ServiceConfig{
+		Name:      util.GetResourceName(p.opssight.Name, util.OpsSightName, name),
+		Namespace: p.opssight.Spec.Namespace,
+		Type:      horizonapi.ServiceTypeServiceIP,
+	})
+
+	service.AddPort(horizonapi.ServicePortConfig{
+		Port:       int32(3007),
+		TargetPort: fmt.Sprintf("%d", 3007),
+		Protocol:   horizonapi.ProtocolTCP,
+		Name:       fmt.Sprintf("port-%s", name),
+	})
+
+	service.AddLabels(map[string]string{"component": name, "app": "opssight", "name": p.opssight.Name})
+	service.AddSelectors(map[string]string{"component": name, "app": "opssight", "name": p.opssight.Name})
+
+	return service
+}
+
+// QuayPerceiverService creates a service for the Quay perceiver
+func (p *SpecConfig) QuayPerceiverService() *components.Service {
+	name := p.names["quay-perceiver"]
+	service := components.NewService(horizonapi.ServiceConfig{
+		Name:      util.GetResourceName(p.opssight.Name, util.OpsSightName, name),
+		Namespace: p.opssight.Spec.Namespace,
+		Type:      horizonapi.ServiceTypeServiceIP,
+	})
+
+	service.AddPort(horizonapi.ServicePortConfig{
+		Port:       int32(3008),
+		TargetPort: fmt.Sprintf("%d", 3008),
```

Comment:
we can keep all the perceiver port as 3002

Reply:
so start server for webhook in my perceiver @ 3002? because right now it's running in 3008

Reply:
we can discuss this tomorrow..